Improve backup location related parts in Synology DSM docs (#37212)
home-assistant/home-assistant.github.io: source/_integrations/synology_dsm.markdown

If you denied access to all locations and applications it is normal to receive a message indicating you do not have access to DSM when trying to login with this separate user. As noted above, you do not need access to the DSM and Home Assistant will still be able to read statistics from your NAS.
{% endnote %}

## Backup location

> The NAS can also be used as a {% term backup %} location, without the need to add the NAS as a network drive to Home Assistant. For this you need to setup the correct permissions for the user (_see [Separate User Configuration](#separate-user-configuration) above_), afterwards you will be able to select the shared folder and define a relative path to be used as backup location in the integration options ({% my integrations title="**Settings** > **Devices & services**" %}) > **Synology DSM** > _select the instance_ > **Configure**)

Reviewer: ```suggestion
The NAS can also be used as a {% term backup %} location, without the need to add the NAS as a network drive to Home Assistant. For this you need to setup the correct permissions for the user (_see [Separate User Configuration](#separate-user-configuration) above_), afterwards, you will be able to select the shared folder and define a relative path to be used as a backup location in the integration options ({% my integrations title="**Settings** > **Devices & services**" %} > **Synology DSM** > _select the instance_ > **Configure**)
```

### Example

Assume there is a shared folder called `HA Backup`, with two directories in it `productive_instance` and `test_instance`.

<img src="/images/integrations/synology_dsm/synology_file_station.png" />

#### Use an existing path

1. Select `HA Backup` as shared folder.
2. Define `productive_instance` as backup path (_without trailing slash_).

>     - The existing `productive_instance` will be used as backup location.

Reviewer: ```suggestion
    - **Result**: The existing `productive_instance` will be used as a backup location.
```

>     - The existing `productive_instance` will be used as backup location.

Author: ```suggestion
    - **Result**: The existing `productive_instance` will be used as backup location.
```

Reviewer: apologies, saw your comment too late

#### Use a non-existing path

1. Select `HA Backup` as a shared folder.
2. Define `cottage_instance` as backup path (_without trailing slash_).

> - **Result**: A new directory `cottage_instance` will be created on the shared folder "HA Backup" during first backup.

Reviewer: ```suggestion
    - **Result**: A new directory `cottage_instance` will be created on the shared folder "HA Backup" during first backup.
```

>     - **Result**: A new directory `cottage_instance` will be created on the shared folder "HA Backup" during first backup.

Reviewer: ```suggestion
    - **Result**: A new directory `cottage_instance` will be created on the shared folder `HA Backup` during the first backup.
```

If you denied access to all locations and applications it is normal to receive a message indicating you do not have access to DSM when trying to login with this separate user. As noted above, you do not need access to the DSM and Home Assistant will still be able to read statistics from your NAS.
{% endnote %}

## Backup location

> The NAS can also be used as a {% term backup %} location, without the need to add the NAS as a network drive to Home Assistant. For this you need to setup the correct permissions for the user (_see [Separate User Configuration](#separate-user-configuration) above_), afterwards you will be able to select the shared folder and define a relative path to be used as backup location in the integration options (**[Settings > Devices & Services](https://my.home-assistant.io/redirect/integrations) > Synology DSM > _select the instance_ > Configure**)

Reviewer: ```suggestion
The NAS can also be used as a {% term backup %} location, without the need to add the NAS as a network drive to Home Assistant. For this you need to setup the correct permissions for the user (_see [Separate User Configuration](#separate-user-configuration) above_), afterwards you will be able to select the shared folder and define a relative path to be used as backup location in the integration options ({% my integrations title="**Settings** > **Devices & services**" %}) > **Synology DSM** > _select the instance_ > **Configure**)
```
your My link was correct, of course, but this way looks prettier, I think :-)

### Example

> Assume there is a shared folder called "HA Backup", with two directories in it "productive_instance" and "test_instance"

Reviewer: ```suggestion
Assume there is a shared folder called `HA Backup`, with two directories in it `productive_instance` and `test_instance`.
```
We use backticks for folder and file names, paths and such.

<img src="/images/integrations/synology_dsm/synology_file_station.png" />

> #### use an existing path

Reviewer: ```suggestion
#### Use an existing path
```

> - select "HA Backup" as shared folder

Reviewer: ```suggestion
1. Select `HA Backup` as shared folder.
```

> - define `productive_instance` as backup path (_without trailing slash_)

Reviewer: ```suggestion
2. Define `productive_instance` as backup path (_without trailing slash_).
```

> 
> The existing `productive_instance` will be used as backup location.

Reviewer: ```suggestion
    - The existing `productive_instance` will be used as backup location.
```

> #### use a non-existing path

Reviewer: ```suggestion
#### Use a non-existing path
```

> - select "HA Backup" as shared folder

Reviewer: ```suggestion
1. Select `HA Backup` as a shared folder.
```

> - define `cottage_instance` as backup path (_without trailing slash_)

Reviewer: ```suggestion
2. Define `cottage_instance` as backup path (_without trailing slash_).
```

> 
> A new directory `cottage_instance` will be created on the shared folder "HA Backup" during first backup.

Reviewer: ```suggestion
- **Result**: A new directory `cottage_instance` will be created on the shared folder "HA Backup" during first backup.
```
